src/style.css

```diff
@@ -1,3 +1,73 @@
-body {
+* {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+html {
+  --active-link: #00acdc;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
+.header img {
+  display: block;
+  margin-left: 50px;
+  padding: 10px 0;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item li {
+  display: flex;
+}
+
+.nav__link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  /* vertical-align: middle; */
+  margin-left: 20px;
+  text-transform: uppercase;
+  text-decoration: none;
+  font-size: 12px;
+  font-weight: 500;
+  /* font-style: normal;
+  line-height: normal; */
+}
+
+a:link {
```

Comment:
If there is page-level styling consider moving it to the body tag at the beginning of your file.
Or you can use a class selector instead